i18n/sv/cosmic_files.ftl

```diff
@@ -57,7 +59,20 @@ paste = Klistra in
 select-all = Välj alla
 
 ## View
+zoom-in = Zooma in
+default-size = Standard storlek
+zoom-out = Zooma ut
```

Comment:
Should the zoom ones be "Förstora" and "Förminska" instead of Zoom based?

Reply:
No hard opinion there, but "Zooma in/ut" is a quite common translation in other places.

I would write "Standardstorlek" as one word though.

Reply:
Fixed that now, sorry for särskrivningen